database/index.js

```diff
@@ -23,8 +23,50 @@ const dbConnection = mysql.createConnection(mysqlConfig);
 
 const getAllExpReviews = (expId, callback) => {
   const query = 'select * from reviews where experience_id = ?';
-  const experienceId = expId;
-  dbConnection.query(query, experienceId, (err, response) => {
+  dbConnection.query(query, expId, (err, response) => {
+    if (err) {
+      callback(err);
+    } else {
+      callback(null, response);
+    }
+  });
+};
+
+const createExpReviews = (expId, reviewsData, callback) => {
+  const reviews = { ...reviewsData };
+  reviews.experience_id = expId;
+  const query = 'INSERT INTO reviews SET ?';
```

Comment:
Does this work? I thought SET was for UPDATE, and this would be INSERT INTO reviews (field, field, ...) VALUES (value, value, ...)

Reply:
Yea I like using this as a reference for mysql queries. Just scroll down a little to see the Insert query

https://www.npmjs.com/package/mysql#escaping-query-values